libs/designer-ui/src/lib/card/__test__/__snapshots__/cardfooter.spec.tsx.snap

```diff
@@ -10,6 +10,8 @@ exports[`lib/card/cardfooter > should render 1`] = `
 </div>
 `;
 
+exports[`lib/card/cardfooter > should render 2`] = `null`;
```

Comment:
Can you update the test or is there a new reason on why its rendering as null?

Reply:
This is just the snapshot so it's generated based on the component, but I adjusted the footer so that it just returns null if there are no badges (less elements to render)